system/CLI/CLI.php

```diff
@@ -262,7 +262,7 @@ public static function prompt(string $field, $options = null, $validation = null
             $default = $options[0];
         }
 
-        static::fwrite(STDOUT, $field . (trim($field) ? ' ' : '') . $extraOutput . ': ');
+        static::fwrite(STDOUT, $field . (trim($field) !== '' && trim($field) !== '0' ? ' ' : '') . $extraOutput . ': ');
```

Comment:
Remove ` && trim($field) !== '0'`. It is kind of bug.

The current behavior.

`What is your favorite color?`:
```console
$ ./spark test

CodeIgniter v4.4.7 Command Line Tool - Server Time: 2024-04-03 22:54:23 UTC+00:00

What is your favorite color? : a
```

`''`:
```console
$ ./spark test

CodeIgniter v4.4.7 Command Line Tool - Server Time: 2024-04-03 22:54:39 UTC+00:00

: a
```

`' '`:
```console
$ ./spark test

CodeIgniter v4.4.7 Command Line Tool - Server Time: 2024-04-03 22:54:48 UTC+00:00

 : a
```

`'0'`:
```console
$ ./spark test

CodeIgniter v4.4.7 Command Line Tool - Server Time: 2024-04-03 22:56:09 UTC+00:00

0: a
```